src/EventSourcedAggregate.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\EventSourcing;
+
+interface EventSourcedAggregate
+{
+    public function apply(object $event): void;
```

Comment:
I guess this is what you're used to in terms of terminology. I've always thought of "applying an event" as using it to respond to it. This is in line with the naming I've used for the aggregate root since the first versions in the 0.x range. I do not have very compelling reasons to change the name, although I've contemplated naming it "when" instead of apply, where "when" also is a little bit more eluding to responding to an event. Then again, one can argue that when($command) would have the same ambiguity associated it with.